homeassistant/components/zwave_js/config_flow.py

```diff
@@ -837,7 +819,7 @@ async def async_step_instruct_unplug(
         # reset the old controller
         try:
             await self._get_driver().async_hard_reset()
-        except FailedCommand as err:
+        except (AbortFlow, FailedCommand) as err:
```

Comment:
I don't think you can get AbortFlow here

Reply:
It's the `_get_driver` call that can raise `AbortFlow`.